batch_import/EXAMPLE.md

```diff
@@ -0,0 +1,210 @@
+  --host 192.168.1.57 \
+  --hook_module "~/examples/batch_import/camio_hooks.py"
+
+hooks module: '/Users/user/examples/batch_import/camio_hooks.py'
+cwd: '/Users/user/examples/batch_import/video-importer'
+camera_name: CAMERA_FRONT
+epoch: 1475971347
+INFO:root:/Users/user/natconv_test/CAMERA_FRONT-rand-1475971347.mp4 (scheduled for upload)
+camera_name: CAMERA_FRONT
+epoch: 1475971947
+INFO:root:/Users/user/natconv_test/CAMERA_FRONT-rand-1475971947.mp4 (scheduled for upload)
+camera_name: CAMERA_FRONT
+epoch: 1475972547
+INFO:root:/Users/user/natconv_test/CAMERA_FRONT-rand-1475972547.mp4 (scheduled for upload)
+camera_name: CAMERA_FRONT
+epoch: 1475973147
+INFO:root:/Users/user/natconv_test/CAMERA_FRONT-003227-1475973147.mp4 (scheduled for upload)
+DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): test.camio.com
+DEBUG:requests.packages.urllib3.connectionpool:https://test.camio.com:443 "POST /api/cameras/discovered HTTP/1.1" 200 3244
+DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): test.camio.com
+DEBUG:requests.packages.urllib3.connectionpool:https://test.camio.com:443 "GET /api/cameras/discovered HTTP/1.1" 200 3402
+Camera ID: u'109010722686218614620:C220161009:81219708c6fe2a5eb9cb35896b8ed78610ce9c6f'
+INFO:root:1/4 uploading /Users/user/natconv_test/CAMERA_FRONT-rand-1475971347.mp4
+Params: {'job_id': None, 'timestamp': '2016-10-08T17:02:27.000', 'uploaded_on': None, 'filename': '/Users/user/natconv_test/CAMERA_FRONT-rand-1475971347.mp4', 'shard_id': None, 'camera': 'CAMERA_FRONT', 'given_name': 'CAMERA_FRONT.2016-10-08T17:02:27.000.4b0bef93c8b4f6a26f5081630b8ad9fb87bb80e0.mp4', 'key': '4b0bef93c8b4f6a26f5081630b8ad9fb87bb80e0', 'discovered_on': '2017-05-05T12:59:12.476759', 'lat': None, 'lng': None, 'confirmed_on': None, 'size': 372157776}
+DEBUG:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): 192.168.1.13
+INFO:root:completed
+INFO:root:2/4 uploading /Users/john/natconv_test/CAMERA_FRONT-rand-1475971947.mp4
+#  ....
+INFO:root:completed
+```
+
+If you get any errors about missing the Device ID of the Camio Box or an unauthenticated error, try to set the environment variables again. To check that the environment variables
+are currently set, you can always `echo $CAMIO_BOX_DEVICE_ID` or `echo $CAMIO_OAUTH_TOKEN` and check that the values printed out match what you expect.
+
+
+#### Seeing the Imported Video
+
+Once the script has completed sending videos to Box for segmentation and first-order analysis, the video clips will begin to be available to search through on [your Camio feed](https://www.camio.com/#feed).
+You can search for the name of the camera (that was parsed from the filenames) to see the videos that have been uploaded.  
+
+We are currently writing some tools to help you recover all of the labels that were generated for the batch-imported videos, but this tool is not available yet. We are also designing tools and an API
```

Comment:
Camio will soon include tools...
The tools will include an API to check on the status...